app/server/__init__.py

```diff
@@ -55,6 +55,7 @@ def create_app():
 
     app.config.from_object('config')
     app.config['BASEDIR'] = os.path.abspath(os.path.dirname(__file__))
+    app.config['EXECUTOR_PROPAGATE_EXCEPTIONS'] = True
```

Comment:
Putting this in `__init__.py` and not in `config.py` or `*_config.ini` files because we _never_ want this to be False

Reply:
🎶 Forever. Forever-never? Forever-never.